app.js

```diff
@@ -31,6 +32,16 @@ client.on('messageDelete', async msg => {
   msg.channel.send(`${author} deleted message: ${message}`);
 });
 
+client.on('presenceUpdate', (oldPresence, newPresence) => {
+  console.log(newPresence);
+  if(oldPresence === undefined || oldPresence.status === newPresence.status) {
+    return;
+  }
+  if(oldPresence.status === 'dnd' || newPresence.status !== 'online') {
+    return;
+  }
+  client.channels.cache.get('962450090117722115').send(newPresence.user.username + ' is online!');
+});
```

Comment:
Nit: console log and could you have return on the same line, sorry just for consistency